### PR TITLE
Update connected accounts panel to show okta identity

### PR DESCRIPTION
**Pull request by cadeParade**
## Description
If a user has an Okta identity, display it on the connected accounts page, but do not allow them to add/remove it.

## Reviewing this PR

_**Is Covered by Automated Tests?: YES**_

### Relevant screenshot: 
[Click here](https://percy.io/percy/percy-web/builds/2587359/view/166366042/1280?browser=chrome&mode=diff)

### CODE REVIEW

File: app/models/identity.js [hunk 1]
```diff
@@ -9,4 +12,12 @@ export default DS.Model.extend({
 
   isGithubIdentity: equal('provider', 'github'),
   isAuth0Identity: equal('provider', 'auth0'),
```
Comment (unicornzero) [inline app/models/identity.js]: Is 'aut0' for user/password identities?

File: tests/factories/identity.js [hunk 2]
```diff
@@ -9,13 +9,15 @@ FactoryGuy.define('identity', {
 
   traits: {
     githubProvider: {
-      provider: () => {
-        return 'github';
-      },
+      provider: 'github',
```
Comment (unicornzero) [inline tests/factories/identity.js]: yay for succinct syntax!

File: app/models/identity.js (hunk 1 above)
Comment (cadeParade) [reply app/models/identity.js]: yes.

File: tests/factories/identity.js (hunk 2 above)
Comment (cadeParade) [reply tests/factories/identity.js]: No need for a function here, since it is returning the same string every time.

**Pull request merged**